src/components/NavigationFolder.vue

```diff
@@ -136,7 +138,7 @@ export default {
 						total: this.folderStats.total,
 					})
 				} else {
-					return n('mail', '{unread} unread of {total}', '{unread} unread of {total}', {
+					return n('mail', '{unread} unread of {total}', '{unread} unread of {total}', undefined, {
```

Comment:
```suggestion
					return n('mail', '{unread} unread of {total}', '{unread} unread of {total}', this.folderStats.total, {
```

I think

Reply:
makes sense, but still the draft sent ect are not loading.. 
[ERROR] mail: could not load folder stats